Documentation/installation/k0s.rst

```diff
@@ -0,0 +1,99 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _k0s_install:
+
+*****************************
+Installation k0s Using k0sctl
+*****************************
+
+This guide walks you through installation of Cilium on `k0s <https://k0sproject.io/>`_,
+an open source, all-inclusive Kubernetes distribution, 
+which is configured with all of the features needed to build a Kubernetes cluster.
+
+Cilium is presently supported on amd64 and arm64 architectures.
+
+Install a Master Node
+=====================
+
+The first step (after installing the k0sctl binary obviously) is to prepare the yaml configuration file k0sctl will use:
+
+.. code-block:: shell-session
+
+   # The following command assumes the user has deployed 3 VMs
+   # with the default user "k0s" using the default ssh-key (without passphrase)
+   k0sctl init --k0s -n "myk0scluster" -u "k0s" -i "~/.ssh/id_rsa" -C "1" 192.168.2.1 192.168.2.2 192.168.2.3 > k0s-myk0scluster-config.yaml
+   
+Next step is editing ``k0s-myk0scluster-config.yaml`` ::
```

Comment:
I think we need to remove the space if we want the colon to show up here. But that's not really important anyway.

```suggestion
Next step is editing ``k0s-myk0scluster-config.yaml``::
```